ros/src/fsds_ros_bridge/include/airsim_ros_wrapper.h

```diff
@@ -205,10 +234,11 @@ class AirsimROSWrapper
     ros::Timer airsim_img_response_timer_;
     ros::Timer airsim_control_update_timer_;
     ros::Timer airsim_lidar_update_timer_;
+    ros::Timer statistics_timer_;

```

Comment:
this timer is responsible for triggering a print and reset of statistics every second (the frequency can be adjusted ofc)